src/main/java/com/example/demo1/repository/OrderRepository.java

```diff
@@ -0,0 +1,17 @@
+package com.example.demo1.repository;
+
+import com.example.demo1.model.Order;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface OrderRepository extends JpaRepository<Order, Long> {
+
+    List<Order> findAllByOrderStatusTrue();
+
```

Comment:
findAllByStatus(String status) (would work if you have a field called status)

Reply:
fixed